extensions/smart-reply/src/types.ts

```diff
@@ -0,0 +1,48 @@
+export interface FormValues {
+  reply: string;
+  tone: string;
+  translationStyle: string;
+}
+
+export interface TranslationResponse {
+  translation: string;
+  detectedLanguage: string;
+}
+
+export interface ReplyResponse {
+  reply: string;
+  retranslation: string;
+}
+
+export type Language =
+  | "ja"
+  | "en"
+  | "fr"
+  | "es"
+  | "de"
+  | "it"
+  | "zh"
+  | "ko"
+  | "pt"
+  | "ru"
+  | "nl"
+  | "vi"
+  | "th"
+  | "id"
+  | "hi"
+  | "ar"
+  | "bn"
+  | "tr"
+  | "uk"
+  | "pl"
+  | "he"
+  | "sv"
+  | "da"
+  | "fi";
+
+export interface Preferences {
+  openAIApiKey: string;
+  targetLanguage: Language;
+  defaultTone: string;
+  defaultStyle: string;
+}
```

Comment:
You don't need to manually set preference types as this is autogenerated in `raycast-env.d.ts`

Reply:
https://github.com/raycast/extensions/commit/21cc1422a60d4dbfd523887d1c56d94dde4b1e29